src/sentry/runner/initializer.py

```diff
@@ -132,9 +132,25 @@ def get_asset_version(settings):
     "mail.use-tls": "EMAIL_USE_TLS",
     "mail.from": "SERVER_EMAIL",
     "mail.subject-prefix": "EMAIL_SUBJECT_PREFIX",
+    "github-login.client-id": "GITHUB_APP_ID",
```

Comment:
@BYK where are these new settings being used?

Reply:
The way we do these migrations is: we introduce the options and deprecate the python settings creating a mapping between them. Then sometime in the future, we remove the python ones when we (and self-hosted users) migrated.